heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/appender/MongoDBAppender.java

```diff
@@ -114,7 +114,7 @@ protected void append(ILoggingEvent e) {
         ZoneId zoneId = ZoneId.of(this.zoneId);
 
         // Offset in milliseconds based on the informed Zone
-        long offset = zoneId.getRules().getOffset(Instant.now()).getTotalSeconds() * 1000;
+        long offset = (long) zoneId.getRules().getOffset(Instant.now()).getTotalSeconds() * 1000;
```

Comment:
Cast is really necessary?

Reply:
Cast removed. Offset variable now is int.